.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
Please add https://github.com/fabric8-hdd/manager-api/pull/2/files#diff-df39304d828831c44a2b9f38cd45289cR12 to suppress console linter errors